.github/workflows/release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Use Node v14
+      - name: Use Node v18
```

Comment:
hahah was about to raise this 😆 

can you find and replace for all the files please?